flyspell-correct.el

```diff
@@ -56,6 +56,36 @@ misspelled word. It has to return either replacement word
 or (command, word) tuple that will be passed to
 `flyspell-do-correct'.")
 
+(defvar  flyspell-direction t
```

Comment:
I remember that you don't like long names and `flyspell-correct` prefix makes it hard to maintain short function names, but let's stick to conventions and use it for now across the package.

Reply:
It seems that `flyspell-direction` is used only in `flyspell-popup-wrapper`. Wouldn't it better to have it inside of that function via `let` binding?